server/src/main/java/fr/uga/l3miage/tp1/exo2/models/NetflixUserEntity.java

```diff
@@ -0,0 +1,35 @@
+package fr.uga.l3miage.tp1.exo2.models;
+
+import fr.uga.l3miage.tp1.exo2.enums.Sex;
+
+import javax.persistence.*;
+import java.math.BigInteger;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "netflix_user")
+public class NetflixUserEntity {
+
+    @Id
+    @Column(name = "id")
+    private BigInteger id;
+
+    @Column(name = "uuid")
+    private String uuid;
+
+    @Column(name = "lastname")
+    private String lastname;
+
+    @Column(name = "firstname")
+    private String firstname;
+
+    @Column(name = "sex", length = 2)
+    private Sex sex;
+
+    @Column(name = "birthDate")
+    private LocalDate birthDate;
+
+    @OneToOne(mappedBy = "netflixUserEntity")
+    private NetflixAccountEntity netflixAccount;
```

Comment:
Alors ici la relation est unidirectionnel normalement, et elle est jointe sur une autre colonne que le @Id donc il faut utiliser le @JoinColumn de cette manière : 
```suggestion
    @OneToOne
    @JoinColumn(name = "uuid_user",referencedColumnName = "uuid")
    private NetflixAccountEntity netflixAccount;
```

Reply:
Oui mais cela n'a jamais été mentionné dans le cours, je l'avais vu en faisant des recherches sur le net (mais puisqu'on peut trouver tout et n'importe quoi, je ne l'ai pas pris en compte)